File: FunnyAlgorithms-master/VolumeOfCylinder/VolumeOfCylinder.c
#include <stdio.h>

int main(void) {
	// your code goes here
	
	float radius,height,volume;
	
	 printf("Enter the radius of the cylinder:");
         scanf("%f",&radius);
        printf("Enter the height of the cylinder::");
         scanf("%f",&height);

         volume = (22 * radius * radius * height) / 7;

        printf("The volume of Cylinder is: %f", volume);


	return 0;
}

File: algorithms/VolumeOfCylinder/volume_of_cylinder.py
import math


def get_cylinder_volume(radius, height):
    return math.pi * radius ** 2 * height


r = float(input("Enter the radius of the cylinder: "))
h = float(input("Enter the height of the cylinder: "))

print("The volume of the cylinder is:", get_cylinder_volume(r, h))
